swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```diff
@@ -0,0 +1,227 @@
+- **Step 03:** Package the compiled Ballerina application into a ZIP file suitable for AWS Lambda deployment.
+- **Step 04:** Ensures that the workflow has the necessary credentials to interact with AWS services.
+- **Step 05:** Deploy the packaged Ballerina function to AWS Lambda by uploading the ZIP file.
+- **Step 06:** Test the deployed Lambda function by invoking it and outputting the response to confirm that the function works as expected.
+
+See [AWS Lambda](https://ballerina.io/learn/aws-lambda/) for more information.
+
+## Azure Functions
+
+In this guide, we'll walk through deploying a serverless application using Azure Functions with Ballerina. 
+You'll learn how to set up, deploy, and manage your application, leveraging the power of Azure's serverless platform 
+to simplify your workflows and eliminate the overhead of server management.
+
+This example demonstrates how to write a simple echo function in Azure Functions using Ballerina, where triggers 
+are represented by listeners attaching an `af:HttpListener` to a service that implies an HTTP trigger. 
+The resource method behaves like a `ballerina/http` service, In the provided code sample, there's an empty service path and a resource path
+named `hello` with a `get` accessor expecting a `name` query parameter, and the `ballerinax/azure_functions` package automatically handles the required artifact generation and data binding.
+
+For more information, see [the Azure deployment guide](https://ballerina.io/learn/azure-functions/).
+
+1. Set up the prerequisites
+For instructions, see [Set up the prerequisites.](https://ballerina.io/learn/azure-functions/#set-up-the-prerequisites)
+
+2. Here is an example of a Ballerina code for the function
+```ballerina
+import ballerinax/azure.functions;
+
+// This function gets triggered by an HTTP call with the name query parameter and 
+// returns a processed HTTP output to the caller.
+service / on new functions:HttpListener() {
+    resource function get hello(string name) returns string {
+        return string `Hello,  ${name}!`;
+    }
+}
+```
+3. Use `bal build` for generating the Azure Function artifacts.
+```
+$ bal build --cloud="azure_functions"
+```
+4. Execute the Azure CLI command given by the compiler to create and publish the functions by replacing <function_app_name> with your respective function app name.
+```
+$ func azure functionapp publish <function_app_name> --script-root target/azure_functions
+```
+5. Execute the command below to invoke the function by replacing <function_app_name> with your respective function app name.
+```
+$ curl https://<function_app_name>.azurewebsites.net/hello\?name\=Jack
+```
+
+### CI/CD with Azure Function
+
+This GitHub Action workflow automates the continuous integration and continuous deployment (CI/CD) process for a Ballerina project.
+It triggers every push to the repository, and builds the project.
```

Comment:
Shouldn't this be ` It is triggered by every push to the repository and automatically builds the project.`?